openapi/main.yml

```diff
@@ -1,38 +1,38 @@
 openapi: 3.0.3
 info:
-  title: Horizon 
+  title: Horizon
   version: 0.0.1
```

Comment:
Should we update this to match the Horizon version string?

Reply:
This is an excellent question! I had the same thought somewhere along the way. I suppose it would open up a-whole-nother conversation about if we want to _keep_ versioned docs for horizon, and how that should be done? [Here's the plugin docs on that](https://docusaurus-openapi.tryingpan.dev/versioning), seems way above my paygrade 🤣